### ISSUE TITLE
Resnet50 pre-trained model, accuracy less than reported

### ISSUE THREAD

**Issue by mojanjp** [keras-team/keras#10040]
I have tried to use the pre-trained weights provided in Keras "applications" to reproduce the results for ImageNet dataset. The images are center-cropped and resized according to the paper. Using model.evaluate I got an accuracy of 74.79% which is less than 75.9% claimed on the website (https://keras.io/applications/). I wonder if I need to perform special image preprocessing on the validation set to get to the 75.9% accuracy. thanks!
